pitchfork/__init__.py

```diff
@@ -1,19 +0,0 @@
-# Copyright 2014 Dave Kludt
```

Comment:
You should never run an application in __init__.py.  There are too many potential issues that are hard to troubleshoot.